cadc-util/src/main/java/ca/nrc/cadc/net/HttpTransfer.java

```diff
@@ -657,7 +657,7 @@ public void setReadTimeout(int readTimeout) {
      */
     public void setDigest(URI checksumURI) {
         String algorithm = checksumURI.getScheme();
-        String checksum = DigestUtil.base64Encode(checksumURI.getSchemeSpecificPart());
+        String checksum = DigestUtil.base64EncodeHex(checksumURI.getSchemeSpecificPart());
```

Comment:
Wouldn't `String checksum = DigestUtil.toDigest(checksumURI);` work here?

Reply:
you mean move this code into DigestUtil so it provides both `getURI(String)` and `toDigest(URI)`? yes, that would be an improvement in clarity. maybe `getDigestHeader(URI)` to be explicit